include/votca/xtp/orbitals.h

```diff
@@ -307,6 +288,14 @@ class Orbitals {
     return _BSE_triplet_coefficients;
   }
 
+  const Eigen::MatrixXd& BSETripletCoefficientsAR() const {
+    return _BSE_triplet_coefficients_AR;
+  }
+
+  Eigen::MatrixXd& BSETripletCoefficientsAR() {
```

Comment:
I guess the answer is yes, but do you need both ?

Reply:
yes, one is the const version the other is not. So yeah